Jenkinsfile

```diff
@@ -1,4 +1,22 @@
 #!/usr/bin/env groovy
+/*
+ * Copyright 2020 Confluent, Inc.
+ *
+ * This software contains code derived from the WePay BigQuery Kafka Connector, Copyright WePay, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
```

Comment:
Do we need a header in this `Jenkinsfile`?

Reply:
It's a mixed bag in our repos; some of them have it, some of them don't. Seems fine to use one here as it has no functional downside. Not sure the verbiage about the software containing "code derived from the WePay BigQuery Kafka Connector, Copyright WePay, Inc." is strictly necessary but to me at least the phrase "this software" refers to this project as a whole and not necessarily this specific source file so I don't think it's inaccurate to include here.

Long story short: don't know if we _need_ a header file, don't think it hurts us to include one though.